syntax = "proto3";

package util_models;

enum PlayerSpecialization {
	Stormtrooper = 0;
	Sentinel = 1;
	Support = 2;
	Medic = 3;
}